docs/src/modules/components/WhatsNewLayout.js

```diff
@@ -168,7 +168,7 @@ const entries = [
     ],
   },
   {
-    title: 'Data Grid v5.15',
+    title: 'MUI X Data Grid v5.15.0',
```

Comment:
Actually, related to https://www.notion.so/mui-org/mui-x-Shorten-Data-Grid-to-Grid-166bffd69b8a47ca9a4f2f45eeb5e4a6